Data/ByteString/Char8.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE MagicHash #-}
 {-# OPTIONS_HADDOCK prune #-}
 {-# LANGUAGE Trustworthy #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
It's a bit sad that there's currently no way to silence just the two specific deprecation warnings we expect.